pkgs/development/libraries/sundials/lapack.nix

```diff
@@ -0,0 +1,33 @@
+{ cmake, fetchurl, python, stdenv, blas, liblapack, gfortran }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sundials";
+  version = "4.1.0";
+
+  src = fetchurl {
+    url = "https://computation.llnl.gov/projects/${pname}/download/${pname}-${version}.tar.gz";
+    sha256 = "19ca4nmlf6i9ijqcibyvpprxzsdfnackgjs6dw51fq13gg1f2398";
+  };
+
+
+  cmakeFlags = [
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+    "-DBLAS_ENABLE=ON" "-DSUNDIALS_INDEX_TYPE=int32_t"
+    "-DBLAS_LIBRARIES=[${blas}/lib/libblas.a,${gfortran.cc.lib}/lib/libgfortran.a]"
```

Comment:
we should see if we can use `.so`/`.dylib` here (and make it more generic via `stdenv.hostPlatform.extensions.sharedLibrary`, instead of `.a`.